src/Components/Publishing/Layouts/NewsLayout.tsx

```diff
@@ -25,45 +30,103 @@ export class NewsLayout extends Component<Props, State> {
 
     this.state = {
       isTruncated: this.props.isTruncated,
+      isHovered: this.props.isHovered,
     }
   }
 
   render() {
-    const { article } = this.props
-    const { isTruncated } = this.state
+    const { article, isMobile } = this.props
+    const { isTruncated, isHovered } = this.state
 
     return (
       <NewsContainer
-        onClick={() =>
-          this.setState({
-            isTruncated: false,
-          })}
         isTruncated={isTruncated}
+        isHovered={isHovered}
+        onMouseEnter={() => {
+          if (!isMobile) {
+            this.setState({ isHovered: true })
+          }
+        }}
+        onMouseLeave={() => {
+          if (!isMobile) {
+            this.setState({ isHovered: false })
+          }
+        }}
```

Comment:
Graduated from using simple CSS `hover` properties to this since we'll need a little more custom behavior to handle mobile that uses viewport to set the hover state: 
![image](https://user-images.githubusercontent.com/2236794/37989665-1a00c442-31d2-11e8-81f2-dcdbcece767b.png)

Reply:
🔥